apps/site/assets/css/_trip-plan-results.scss

```diff
@@ -24,10 +24,16 @@
     background-color: $brand-primary-lightest-contrast;
     border: 1px solid $brand-primary;
     border-bottom: 0;
+    display: flex;
     font-size: $base-spacing;
+    justify-content: space-between;
     line-height: 1.5;
     margin-top: 0;
     padding: $base-spacing;
+
+    @include media-breakpoint-up(sm) {
```

Comment:
It's backwards! media-breakpoint-**down** will make it apply to sm and smaller :)

Reply:
:woman_facepalming: